doc/075_scripting.rst

```diff
@@ -722,6 +722,30 @@ The stats command returns a single JSON object.
 | ``compression_space_saving`` | Overall space saving due to compression             |
 +------------------------------+-----------------------------------------------------+
 
+tag
+---
+
+The ``tag`` command uses the JSON lines format with the following message types.
+
+Changed
+^^^^^^^
+
++--------------------------+-------------------------------------------+
+| ``message_type``         | Always "changed"                          |
++--------------------------+-------------------------------------------+
+| ``original_snapshot_id`` | ID of the snapshot before the change      |
```

Comment:
After discussion with @rawtaz, let's rename this to `old_snapshot_id`. The term original is overloaded in the context of the `tag` command as command also sets a field `original` when first modifying a snapshot.

Reply:
Done. I have renamed this to `old_snapshot_id` now.